### PR TITLE
Aws/mail

### PR DESCRIPTION
**Pull request by satu0king**
1. Adds mailkick gem
2. Adds AWS-SES Configuration

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -83,18 +83,19 @@
...
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = {
+  #   :user_name => 'apikey',
+  #   :password => ENV["CIRCUITVERSE_EMAIL_PASSWORD"],
+  #   :domain => 'circuitverse.org',
+  #   :address => 'smtp.sendgrid.net',
+  #   :port => 587,
+  #   :authentication => :plain,
+  #   :enable_starttls_auto => true
+  # }
```
Comment (satu0king) [inline config/environments/production.rb:98]: Commented out send-grid creds for now. Should I remove it?

**Comment by satu0king** [comment]
some weird bug regarding `host` is causing tests to fail. @tachyons might know how to fix this.

File: db/schema.rb [hunk 2]
```diff
@@ -338,12 +351,18 @@
     t.string "uid"
     t.string "profile_picture_file_name"
     t.string "profile_picture_content_type"
-    t.bigint "profile_picture_file_size"
+    t.integer "profile_picture_file_size"
```
Comment (tachyons) [inline db/schema.rb:354]: How is this changed ?

File: db/schema.rb [hunk 3]
```diff
@@ -338,12 +351,18 @@
...
     t.string "country"
     t.string "educational_institute"
-    t.boolean "subscribed", default: true
+    t.boolean "subscribed", default: false
+    t.boolean "accepted_privacy_policy", default: false
+    t.string "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string "unconfirmed_email"
+    t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
```
Comment (tachyons) [inline db/schema.rb:365]: These fields are from unmreged PRs, right ?

File: db/schema.rb (hunk 2 above)
Comment (satu0king) [reply db/schema.rb:354]: I don't know 😢

File: db/schema.rb (hunk 3 above)
Comment (satu0king) [reply db/schema.rb:365]: Yeah, I will reset this.

File: db/schema.rb (hunk 2 above)
Comment (satu0king) [reply db/schema.rb]: Fixed

File: config/environments/production.rb (hunk 1 above)
Comment (tachyons) [reply config/environments/production.rb:98]: why not

**Review by tachyons — APPROVED** [review]

**Comment by satu0king** [comment]
Merged indirectly

**Pull request closed without merging**